src/features/joinForms/panes/JoinSubmissionPane.tsx

```diff
@@ -41,43 +41,75 @@ const JoinSubmissionPane: FC<Props> = ({ orgId, submissionId }) => {
     }
   }
 
+  const Header = () => {
+    if (data.accepted) {
+      return (
+        <Box marginBottom={2} marginRight={2}>
+          <NextLink
+            href={`/organize/${orgId}/people/${data.person_data.id}`}
+            legacyBehavior
+            passHref
+          >
+            <Link style={{ cursor: 'pointer' }} underline="none">
+              <Box alignItems="center" display="flex">
+                <Avatar
+                  src={`/api/orgs/${orgId}/people/${data.person_data.id}/avatar`}
+                  style={{ height: 30, width: 30 }}
+                />
+                <Box
+                  alignItems="start"
+                  display="flex"
+                  flexDirection="column"
+                  justifyContent="center"
+                  ml={1}
+                >
+                  <Typography fontSize={30} variant="h3">
+                    {`${data.person_data.first_name} ${data.person_data.last_name}`}
+                  </Typography>
+                </Box>
+              </Box>
+            </Link>
+          </NextLink>
```

Comment:
I think it might have been possible to just put this in the `title` of the `PaneHeader`, but this is fine too!